code/modules/mob/living/simple_animal/borer.dm

```diff
@@ -223,6 +223,10 @@ var/global/borer_unlock_types = typesof(/datum/unlockable/borer) - /datum/unlock
 
 		stat("Chemicals", chemicals)
 
+/mob/living/simple_animal/borer/start_pulling(var/atom/movable/AM)
+	to_chat(src, "<span class='warning'>You are too small to pull anything.</span>")
+	return
```

Comment:
I mean, this is exactly how mouse does it.

Would you rather I make another mob var and have it checked in the parent for start_pulling?